channels/eg.m3u

```diff
@@ -89,8 +89,6 @@ http://95.170.215.109/hls/m3u8/Pan-Drama-Backup.m3u8
 http://95.170.215.109/hls/m3u8/Panorama2-Backup.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="Food",Panorama Food
 http://95.170.215.107/hls/m3u8/Pano-Food-Backup.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/A0O9wUD.png" group-title="",Rotana Masriya
```

Comment:
Remove Rotana Masriya as it's been renamed to Rotana Drama (which is in sa.m3u)

https://en.wikipedia.org/wiki/Rotana_Masriya